src/meetings/2021-09-20.md

```diff
@@ -1,11 +1,11 @@
 # 2021-09-20
 
-| Info                                           |                          |
-| ---------------------------------------------- | ------------------------ |
-| Time                                           | [See in your time zone]  |
-| Calendar event (with Zoom details)             | [link][cal]              |
-| Social hour calendar event (with Zoom details) | [link][calsh]            |
-| Recording                                      | (link posted after call) |
+| Info                                           |                                      |
+| ---------------------------------------------- | -------------------------------------|
+| Time                                           | [See in your time zone]              |
+| Calendar event (with Zoom details)             | [link][cal]                          |
+| Social hour calendar event (with Zoom details) | [link][calsh]                        |
+| Recording                                      | [link](https://youtu.be/ruStOudV2T0) |
```

Comment:
I think it would make sense to have this as a link below, like the rest of the calendar/time links

Reply:
Good idea!

Could you please take another look?